include/sensr_message_listener.h

```diff
@@ -23,7 +23,8 @@ namespace sensr {
     MessageListener(ListeningType listening_type = ListeningType::kOutputMessage);
     virtual ~MessageListener() = default;
     virtual void OnError(Error error, const std::string& reason);
-    virtual void OnGetOutpuMessage(const sensr_proto::OutputMessage &message);
+    virtual void OnGetOutpuMessage(const sensr_proto::OutputMessage &message); // Deprecated!!!!
```

Comment:
Can we use the [[deprecated]] attribute for this?

Reply:
Ah, actually, I was trying to use it but [[deprecated]] is possible from c++14. :cry: